main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,6 +1,7 @@
 {
   "methods": {
     "java.lang.Boolean::getBoolean": "Env, IO",
+    "java.lang.Double::resolveConstantDesc": "Sys, IO",
```

Comment:
What on God's green earth is this.

Reply:
Same as https://github.com/flix/flix/pull/9646